Added a suggestion in Setup Linking

frontegi opened the pull request:
Just spent 2 hours on checking every configuration (AWS, Alexa Dev, Home Assistant logs...) , then figured out that active IP BAN was preventing to continue with the linking.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

frontegi committed:
Added a suggestion in Setup Linking

Just spent 2 hours on checking every configuration (AWS, Alexa Dev, Home Assistant logs...) , then figured out that active IP BAN was preventing to continue with the linking.

homeassistant commented:
Hi @frontegi, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22660).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck commented on source/_integrations/alexa.smart_home.markdown:
```diff
@@ -217,6 +217,7 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - Click the Smart Home skill you just created.
   - Click `Enable to use`.
   - A new window will open to direct you to your Home Assistant's login screen.
+    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
```
I'm not sure if this comment is clear or what the intention is?

Right now it reads like: You need to activate it?

Additionally, it doesn't matter if IP Ban is activated or not?

frontegi replied:
Hi @frenck , you are right, statement not completely clear. The problem is that after some tests, activated IP ban could prevent the conclusion of the procedure. I will try to better explain it.

frontegi committed:
Account Linking final stage,IP Ban issue clarified

Modified to a more clear statement.
Included a link to the HTTP IP Ban documentation.

frontegi reviewed (commented):
Now the statement seems to be more clear.

frenck commented on source/_integrations/alexa.smart_home.markdown:
```diff
@@ -217,7 +217,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - A new window will open to direct you to your Home Assistant's login screen.
-    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
+    
+    note: If you have [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) activated, after many try, there is the possibility receive a "403 Forbidden" return code from the Home assistant's login screen.
```
So, how would many tries happen?

Some additional notes:

- Start sentences with capital letters.
- Don't use full URLs, instead, make URLs from the website root.

frenck commented on source/_integrations/alexa.smart_home.markdown:
```diff
@@ -217,7 +217,9 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
-    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
+    
+    note: If you have [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) activated, after many try, there is the possibility receive a "403 Forbidden" return code from the Home assistant's login screen.
+    In this case, temporarily disable [IP Ban](https://www.home-assistant.io/integrations/http/#ip_ban_enabled) to conclude the procedure or remove the banned IP entry from the list.
```
Let's not recommend disabling it.

frontegi replied:
Right. sorry, learning how to proactively contribute ! :-)

frontegi committed:
Removing unsecure suggestion

As suggested, removing a suggestions that can potentially expose the HA installation to attackers.
Added link to the blocked list feature documentation.

frontegi reviewed (commented):
As suggested, removing a suggestions that can potentially expose the HA installation to attackers.
Added link to the blocked list feature documentation.
